adoc/extensions/sycl_khr_queue_empty_query.adoc

```diff
@@ -32,6 +32,10 @@ below.
 This extension adds the following function to the [code]#sycl::queue# class,
 which provides information about the emptiness of the queue.
 
+{note} This khr is more usefull when used in conjuction with
+[api]#queue::khr_flush()#.
```

Comment:
```suggestion
[api]#queue::khr_flush#.
```

This must exactly match the `[apidef]`.  Also, I'm moving to a style where we refer to function names without the `()`.  This is the same style as cppreference.

I'm surprised this does not result in a build error, I thought the scripting used to generate an error if an `[api]` did not have a matching `[apidef]`.  I wonder if that error checking logic got broken somehow.

Reply:
Look like the implementation is here`api_xrefs/extension.rb`
And indeed, without looking to closely I was expecting
```
        # Make sure there was a definition for this API.  Diagnose an error if
        # not.
        if not api_to_id.key?(api)
          logger.error "[api]##{api}#: Not defined"
 ```
 To fail

Reply:
> But I think it's because `()` doesn't match the regex (`[\w:]+`), so they were not considered API at all. I think that `khr_flush2` would have triggered an error

Indeed.  Looks like this has been a bug since I first added this extension.  I'll try to take a look later today.